compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
it would be better to split this script and Docker files.
or can you explain the why Dockerfile and this file has close relation so that should be in one PR?

Reply:
Yes. Dockerfile is not related with this script and I'll remove Dockerfile on next PR
We want to show people complete Dockerfile that this script will make but It was wrong.

Reply:
> We want to show people complete Dockerfile that this script will make but It was wrong.

In this case, you can use a draft PR. A Draft PR is not for merging, so you can show whole files you want. :)